NPE in ObaRoutesForLocationRequest.Builder

Haven't gotten to investigate this. It only happened 5 times this week.

java.lang.RuntimeException: An error occured while executing doInBackground()
at android.support.v4.content.ModernAsyncTask$3.done(ModernAsyncTask.java:137)
at java.util.concurrent.FutureTask$Sync.innerSetException(FutureTask.java:274)
at java.util.concurrent.FutureTask.setException(FutureTask.java:125)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:308)
at java.util.concurrent.FutureTask.run(FutureTask.java:138)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1088)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:581)
at java.lang.Thread.run(Thread.java:1019)
Caused by: java.lang.NullPointerException
at com.joulespersecond.oba.request.ObaRoutesForLocationRequest$Builder.<init>(ObaRoutesForLocationRequest.java:40)
at com.joulespersecond.seattlebusbot.MySearchRoutesFragment$MyLoader.loadInBackground(MySearchRoutesFragment.java:227)
at com.joulespersecond.seattlebusbot.MySearchRoutesFragment$MyLoader.loadInBackground(MySearchRoutesFragment.java:1)
at android.support.v4.content.AsyncTaskLoader.onLoadInBackground(AsyncTaskLoader.java:240)
at android.support.v4.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:51)
at android.support.v4.content.AsyncTaskLoader$LoadTask.doInBackground(AsyncTaskLoader.java:40)
at android.support.v4.content.ModernAsyncTask$2.call(ModernAsyncTask.java:123)
at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:306)
... 4 more

--- a/src/com/joulespersecond/oba/region/ObaRegionsTask.java
+++ b/src/com/joulespersecond/oba/region/ObaRegionsTask.java
@@ -165,6 +165,7 @@ public class ObaRegionsTask extends AsyncTask<Void, Integer, ArrayList<ObaRegion
 
         AlertDialog.Builder builder = new AlertDialog.Builder(mContext);
         builder.setTitle(mContext.getString(R.string.region_choose_region));
+        builder.setCancelable(false);
         builder.setItems(items, new DialogInterface.OnClickListener() {
 
             public void onClick(DialogInterface dialog, int item) {

